_compdemos/first_rhino.md

```diff
@@ -0,0 +1,222 @@
+please ensure you are connected to the Fred Hutch network using one of the options described in [off campus log-in](#off-campus-log-in).
+
+In your shell,
+type the following (where `username` is your HutchNetID):
+
+    ssh username@rhino
+
+Hit enter to execute the command.
+The first time you execute this on a computer,
+you'll receive a response similar to:
+
+```
+The authenticity of host 'rhino.fhcrc.org' can't be established.
+Are you sure you want to continue connecting (yes/no)?
+
+Type `y` and press enter.
+```
+
+You'll be prompted to enter your username and then password
+(note that your cursor will not move as you type your password; this is normal!).
+
+After you have successfully logged in,
+you'll see the information about the cluster printed to your screen.
+You'll be ready to start entering commands when you see a prompt like the following appear:
+
+    username@rhino2:~$
+
+## Logging on outside of the Fred Hutch network
+
+Logging in to `rhino` as described above assumes you are already connected to the Fred Hutch network.
+If you are off campus,
+you will need to first connect to the campus network.
+Here we describe the use of `snail` for remote login,
+since it doesn't require the use of additional software beyond what it already included in the tutorial.
+VPN may be a preferrable option for you,
+especially if you are already using it to access other Hutch resources.
+For more information about remote login, please see [Access via a remote login](https://sciwiki.fredhutch.org/scicomputing/access_methods/#access-via-a-remote-location).
+
+In your shell,
+type the following (username is your HutchNetID):
+
+    ssh username@snail.fhcrc.org
+
+You'll see a message printed to the screen that starts with:
+
+```
+Welcome to the Fred Hutchinson Cancer Research Center
+```
+
+Your command prompt will now look something like:
+
+    [username@ssh-gw1-prd ~]$
+
+Because you've already logged on to `snail` with your username,
+you can use the following shortcut to log on to `rhino`:
+
+    ssh rhino
+
+Follow the process as [described above](#within-the-fred-hutch-network)
+to enter your password and access `rhino`.
+
+## Logging on to `gizmo` via `grabnode`
+
+When you log on to the cluster,
+all commands you run are limited by the compute resources associated with your default login
+(and by the cluster being shared by many people!).
+It's good etiquette, and often a necessity,
+to request your own node for additional tasks.
+
+You can accomplish this by executing the following command:
+
+    grabnode
+
+You'll now be asked a series of questions about what compute resources you would like to request.
+We'll describe requesting the smallest unit of resources here for demonstration purposes.
+When prompted:
+
+- type 1 for number of CPUs
+- hit `Enter` to select the default for memory
+- type 1 for number of days
+
+When requesting resources for your own work,
+choose appropriate selections to suit the needs of the task.
+
```

Comment:
Give an example of what other options users have, such as num of nodes, num of CPU's, time to run, etc.; follow with an example of the command on the command line